gf-guide/article.md

```diff
@@ -0,0 +1,139 @@
+
+# Article and Description files
+{:.no_toc}
+
+<div class="callout">
+
+🦉 Both the <mark class="grey">ARTICLE.en_us.html</mark> and <mark class="grey">DESCRIPTION.en_us.html</mark> files are used to bring the information that appears in the <code>#About</code> section on any specimen page of <a href="https://fonts.google.com">Google Fonts</a> to provide further information about the font family.
+<br><br>
+Any of these files will be created by the team member onboarding the font. Thus, the actual file a designer should write carefully is the <a href="./readmefile">README.md</a> file of the font's repository as the information in it will be crucial create a descriptive <a href="https://fonts.google.com/specimen/Kalnia#about">#About</a> section.
+
+</div>
+
+<div class="context-reading">
+    Background reading:<br>
+    <mark class="green"><b>must&rarr;</b></mark> <a href="./readmefile" style="font-weight:bold">README file</a>
+</div>
+
+## Table of contents
+{:.no_toc}
+* TOC goes here
+{:toc}
+
+## The Article
+
+The Article is a new expanded version of the `DESCRIPTION.en_us.html` file that includes images and longer-form information and content. It appears on the Google Fonts site as the main content in the About & License section. For example, see the About page of [Playpen Sans](https://fonts.google.com/specimen/Playpen+Sans/about), and then view the `ofl/playpensans` directory in the Google Fonts repo: [github.com/google/fonts/tree/main/ofl/playpensans](https://github.com/google/fonts/tree/main/ofl/playpensans). 
+
+The `ARTICLE.en_us.html` file and images should live in an `article/` directory alongside the `METDATA.pb` file in the root of the `ofl/fontname` directory. Typically, only the Noto fonts are expected to combine the `DESCRIPTION.en_us.html` and `ARTICLE.en_us.html` files. For all the other families (non-Noto), if an article with images is added, the description should be overwritten by the Article. This means we should shift all the Description content into the Article and delete the `DESCRIPTION.en_us.html`.
+
+Here are some guidelines for how to prepare the `article/` directory and files:
+
+### Requirements
+
+- **Pay attention to the content** Feel free to write a comprehensive text that will pay tribute to your project. Here are added examples of 'About' sections on Google Fonts. <a href="https://fonts.google.com/specimen/Ojuju/about"> Ojuju</a> and <a href="https://fonts.google.com/specimen/Roboto+Flex/about"> Roboto Flex</a>.
+- **Should be written in the third person**
+- **Copywriting should not be inappropriate, stereotypical, or offensive.** For example, "This family demonstrates a feminine energy with its fragile aspect" is unacceptable.
+- **The text length** should be around 500 words, and more than 100 characters.
+- **If the font is derived from another font** should mention the original with a URL to the font on GF if it exists.
```

Comment:
Add colon or "If the font is derived from another font. It should mention the original with a URL to the font on GF if it exists."

Reply:
Solved